crates/runner-host/Cargo.toml

```diff
@@ -20,6 +20,7 @@ wasefire-board-api = { path = "../board", features = ["std"] }
 wasefire-error = { path = "../error" }
 wasefire-interpreter = { path = "../interpreter", optional = true }
 wasefire-logger = { path = "../logger" }
+wasefire-cli-tools = {path = "../cli-tools"}
```

Comment:
CI should complain about ordering and formatting here.

Reply:
Fixed with formatter